core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -221,8 +221,8 @@ public NativeImageBuildItem build(NativeConfig nativeConfig, NativeImageSourceJa
                 nativeConfig.enableAllSecurityServices = true;
             }
 
-            nativeConfig.additionalBuildArgs.ifPresent(l -> l.stream().map(String::trim).forEach(command::add));
             command.add("--initialize-at-build-time=");
+            nativeConfig.additionalBuildArgs.ifPresent(l -> l.stream().map(String::trim).forEach(command::add));
```

Comment:
What's the rationale behind this? Not that it's particularly important :)

Reply:
Ah sorry, it shouldn't have been there. I removed it.

Reply:
Thanks